tools/sctool/src/main/java/com/horizen/ScBootstrappingToolCommandProcessor.java

```diff
@@ -502,6 +503,17 @@ private void processGenesisInfo(JsonNode json) {
             return;
         }
 
+        boolean isNonCeasing = false;
+        int virtualWithdrawalEpochLength = 0;
+        if (json.has("virtualWithdrawalEpochLength") && json.get("virtualWithdrawalEpochLength").isInt()) {
```

Comment:
I think, we need to check if `virtualWithdrawalEpochLength` is present but not `Int` -> error